Add CLI commands to extract package versions and show rendered inventory (#567)
projectsyn/commodore: docs/modules/ROOT/pages/reference/commands.adoc

The option `--alias` (or `-a`) can be used to compile an instance-aware component with a specific instance.

== Inventory Show

  commodore inventory show|components|packages GLOBAL_CONFIG [TENANT_CONFIG]

The command `commodore inventory show` prints the `parameters` in the Commodore hierarchy in directory `GLOBAL_CONFIG`.
If provided, the command also takes into account the tenant repo in directory `TENANT_CONFIG`.

> The commands `commodore inventory components` and `commodore inventory packages` also render the inventory, but only print a list of all components or packages.

Reviewer: This feels more natural to me, but I'm not sure if it's actually more correct:

```suggestion
The commands `commodore inventory components` and `commodore inventory packages` also render the inventory, but only print the list of all components and packages respectively.
```
